assets/css/styles.css

```diff
@@ -1241,3 +1241,13 @@ a {
     left: 18%;
   }
 }
+
+/*img class is to transform image in bigger when move the cursor on image */
+img {
+    transition:transform 0.25s ease;
+}
+
+img:hover {
```

Comment:
This also has to be `.img` to get the intended result. This applies the transform to all images on the site. Please visit https://deploy-preview-146--cranky-borg-621833.netlify.com/2019/ to get an idea of what the end result is 😆 

I would suggest naming this something like `.img-zoom` so that it doesn't cause any unintentional side-effect.

Reply:
Modified to .img-zoom in css and modified the class name to img-zoom in markdown file.